plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -270,7 +270,11 @@ func (p *pusher) send() {
 				p.Log.Errorf("Unable to create log stream %v/%v: %v", p.Group, p.Stream, e.Message())
 			}
 		case *cloudwatchlogs.InvalidSequenceTokenException:
-			p.Log.Warnf("Invalid SequenceToken used, will use new token and retry: %v", e.Message())
+			if p.sequenceToken == nil {
+				p.Log.Infof("First time sending logs to %v/%v since last restart so sequenceToken is nil, learned new token:(%v): %v",  p.Group, p.Stream, e.ExpectedSequenceToken, e.Message())
```

Comment:
I think the only thing I disagree with is the wording regarding restarts. We would reach this log the first time we try to push for any stream, right? Having the word "restart" in this log might confuse someone into thinking that the agent restarted, when that isn't always the case.

Reply:
Ok, changed "since last restart" to "since startup". Hope that is more clear. If not, please provide an alternate wording.